From 1dde6b3c32177ef487f019e579744339eb3e2115 Mon Sep 17 00:00:00 2001
From: Dan Petrisko <petrisko@cs.washington.edu>
Date: Fri, 17 May 2024 12:15:22 -0500
Subject: [PATCH] Using yosys.tcl

---
 project.py       | 23 +++++++++++------------
 requirements.txt |  2 +-
 2 files changed, 12 insertions(+), 13 deletions(-)

diff --git a/project.py b/project.py
index 1441bda..0c898d7 100644
--- a/project.py
+++ b/project.py
@@ -154,6 +154,12 @@ class Project:
         yosys_cmd = 'yowasp-yosys -qp "$YOSYS_CMD"'
         return subprocess.run(yosys_cmd, shell=True, env=env, capture_output=no_output)
 
+    def run_yosys_tcl(self, no_output: bool = False):
+        env = os.environ.copy()
+        script = "$PROJ_ROOT/src/yosys.tcl"
+        yosys_cmd = 'yosys -c $PROJ_ROOT/src/yosys.tcl'
+        return subprocess.run(yosys_cmd, shell=True, env=env, capture_output=no_output)
+
     def check_ports(self, include_power_ports: bool = False):
         top = self.get_macro_name()
         if not self.is_user_project and self.is_chip_rom():
@@ -161,21 +167,14 @@ class Project:
         sources = [os.path.join(self.src_dir, src) for src in self.sources]
         source_list = " ".join(sources)
 
-        json_file = "ports.json"
-
-        # Heuristic - try reading just the first source file, if that fails, try all of them
-        p = self.run_yosys(
-            f"read_verilog -lib -sv {sources[0]}; hierarchy -top {top} ; proc; write_json {json_file}",
-            True,
-        )
-        if p.returncode != 0:
-            p = self.run_yosys(
-                f"read_verilog -lib -sv {source_list}; hierarchy -top {top} ; proc; write_json {json_file}"
-            )
+        p = self.run_yosys_tcl()
         if p.returncode != 0:
-            logging.error(f"yosys port read failed for {self}")
+            logging.error(f"yosys script failure")
             exit(1)
 
+        json_file = "ports.json"
+        shutil.copyfile("output.json", json_file)
+
         with open(json_file) as fh:
             ports = json.load(fh)
         os.unlink(json_file)
diff --git a/requirements.txt b/requirements.txt
index 4a13b99..5394012 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -8,7 +8,7 @@ chevron==0.14.0
 click==8.1.3
 cssselect2==0.7.0
 defusedxml==0.7.1
-gdstk==0.9.49
+gdstk==0.9.51
 gitdb==4.0.10
 GitPython==3.1.40
 idna==3.4
-- 
2.34.1

