diff --git a/project.py b/project.py
index 1441bda..2e0e7d2 100644
--- a/project.py
+++ b/project.py
@@ -288,7 +288,7 @@ class Project:
         return Repo(self.local_dir).commit().hexsha
 
     def get_tt_tools_version(self):
-        repo = Repo(os.path.join(self.local_dir, "tt"))
+        repo = Repo(os.path.join(self.local_dir, "../tt-support-tools"))
         return f"{repo.active_branch.name} {repo.commit().hexsha[:8]}"
 
     def read_commit_info_json(self) -> typing.Dict[str, typing.Any]:
